files/build_templates/sonic_debian_extension.j2

```diff
@@ -359,7 +359,10 @@ sudo chmod 755 $FILESYSTEM_ROOT/usr/bin/restart_service
 
 # Installed smartmontools version should match installed smartmontools in docker-platform-monitor Dockerfile
 # TODO: are mismatching versions fine for bookworm?
-sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install smartmontools
+echo "deb http://deb.debian.org/debian bookworm-backports main" | sudo tee -a $FILESYSTEM_ROOT/etc/apt/sources.list > /dev/null
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get update
```

Comment:
@k-v1 your suggestion is not working 

![image](https://github.com/sonic-net/sonic-buildimage/assets/45705344/eb724bfe-2d3a-42f0-8060-cf2ce343a1b7)


Here is the latest build failure :- https://[dev.azure.com/mssonic/be1b070f-be15-4154-aade-b1d3bfb17054/_apis/build/builds/457492/logs/58](https://dev.azure.com/mssonic/be1b070f-be15-4154-aade-b1d3bfb17054/_apis/build/builds/457492/logs/58)

Reply:
Looks like the version upgrade pipeline has been failing on master branch at least since October, because Barefoot is failing to build.

@xumia @lguohan can we disable the Barefoot build for 202311 and master, considering that platform has been failing to build for months? Looks like the SAI library for that platform needs to be updated.